README.md

```diff
@@ -1 +1,110 @@
-# predictive-market-cameligo
\ No newline at end of file
+# Betting Contract
+
+This contract represents a Tezos contract written in CameLIGO in which users can bet on events added by the Admin or an Oracle.
+
+The current implementation of the contract is as follows :
+
```

Comment:
Can you provide a workflow doc like you provided for dao https://github.com/ligolang/dao-cameligo/blob/main/docs/30-specification.md

Reply:
Done 👍🏻